starter-code/package.json

```diff
@@ -6,17 +6,19 @@
     "start": "nodemon ./bin/www"
```

Comment:
Better add at least two scripts, one for start (with node) and another for development (with nodemon).